src/renderer/components/outputs.tsx

```diff
@@ -0,0 +1,87 @@
+import { observer } from 'mobx-react';
```

Comment:
Question: What's the purpose of the `Outputs` wrapper component?

Reply:
Outputs abstracts a lot of the monaco setup/loading behaviour- I made this class after needing a way to pass in monaco and monaco options to output.tsx